ssv/runner.go

```diff
@@ -93,15 +93,19 @@ func (b *BaseRunner) basePreConsensusMsgProcessing(runner Runner, signedMsg *Sig
 }
 
 func (b *BaseRunner) baseConsensusMsgProcessing(runner Runner, msg *qbft.SignedMessage) (decided bool, decidedValue *types.ConsensusData, err error) {
-	if err := b.validateConsensusMsg(msg); err != nil {
-		return false, nil, errors.Wrap(err, "invalid consensus message")
+	prevDecided := false
+	if b.HasRunningDuty() && b.State != nil && b.State.RunningInstance != nil {
+		prevDecided, _ = b.State.RunningInstance.IsDecided()
 	}
 
-	prevDecided, _ := b.State.RunningInstance.IsDecided()
-
 	decidedMsg, err := b.QBFTController.ProcessMsg(msg)
 	if err != nil {
-		return false, nil, errors.Wrap(err, "failed to process consensus msg")
+		return false, nil, err
```

Comment:
why removed wrapping?

Reply:
We wrap the err with this string also in the runner level (e.g. attester)